core/chain/abci.go

```diff
@@ -233,3 +274,35 @@ func (app *CoreApplication) isValidProtoEvent(tx []byte) (*gen_proto.Event, erro
 	}
 	return &msg, nil
 }
+
+func (app *CoreApplication) isValidSlaRollup(tx []byte) bool {
+	var msg gen_proto.SlaRollup
+	err := proto.Unmarshal(tx, &msg)
+	return err == nil
+}
+
+func (app *CoreApplication) validateBlockTxs(ctx context.Context, blockTime time.Time, blockHeight int64, txs [][]byte) (bool, error) {
+	alreadyContainsRollup := false
+	app.startInProgressTx(ctx)
```

Comment:
For validation methods, let's not write any data to the db. I wouldn't expect that just by reading the name. What do we need this inprogress tx for here?